color/color_test.go

```diff
@@ -0,0 +1,79 @@
+// +build linux darwin
+
+package multicolor
+
+import (
+	"fmt"
+
+	"github.com/ghishadow/color"
+)
+
+func ExampleNew() {
+	// New is very flexible.
+
+	// It accepts an enumeration of attributes.
+	New(color.FgGreen, color.Underline)
+
+	// Or an array of attributes.
+	var attributes []color.Attribute = Attributes(color.FgGreen, color.Underline)
+	New(attributes)
+
+	// As well as an enumeration of strings.
+	New("Green", "Underline")
+
+	// It can also mix strings and attributes
+	New(color.BgRed, "crossedout")
+
+	// Attributes can also be separated by spaces.
+	New("Green Underline")
+
+	// Or separated by any non letter. Case is also non significant.
+	New("RED | underline, CrossedOUT+BgYellow")
+
+	// But it panics if you supplied invalid attribute.
+	func() {
+		defer func() { fmt.Println(recover()) }()
+		New("red, invalid | rouge")
+	}()
+	// Output:
+	// Attribute not found invalid
```

Comment:
These examples would be nicer if they were split into multiple ones. Currently, it doesn't show that the attributes were actually parsed. Just that there's no error.

Or just something like this:
fmt.Println("Creating a writer with color's attributes")
writer := New(color.FgGreen, color.Underline)
fmt.Println(writer.Color)

Then your output would make sense and we would know what example gives what output